documentation/ipi-install/modules/ipi-install-config-yaml.adoc

```diff
@@ -24,6 +24,8 @@ platform:
     dnsVIP: <dns-ip>
     provisioningNetworkInterface: <NIC1>
     provisioningNetworkCIDR: <CIDR> # i.e. fd00:1101::1/64
+    bootstrapOSImage: http://<Disconnected-Registry-fqdn>/rhcos-qemu-sha256
+    clusterOSImage: http://<Disconnected-Registry-fqdn>/rhcos-openstack-sha256 
     provisioningBridge: provisioning
     hosts:
       - name: openshift-master-0
```

Comment:
The fix isn't this simple. These pieces are optional. So they wouldn't not be part of the install-config.yaml directly. They would be part of the https://openshift-kni.github.io/baremetal-deploy/4.4/Deployment.html#additional-install-config-parameters_ipi-install-prerequisites , but in actuality they would be part of this PR which isn't merged yet: https://github.com/openshift-kni/baremetal-deploy/pull/418